.azure-pipelines/generate-beta-modules.yml

```diff
@@ -6,23 +6,24 @@ name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
 trigger:
   branches:
     include:
-    - master
+      - master
   paths:
     include:
-    - src/Beta/*
-    - config/ModulesMapping.jsonc
+      - src/*
+      - config/ModulesMapping.jsonc
+    exclude:
+      - src/Authentication/*
 pr: none
 variables:
   MODULE_PREFIX: 'Microsoft.Graph'
-  WORKLOAD_MODULE_PATH: 'src\beta\'
-  GRAPH_VERSION: 'beta'
+  WORKLOAD_MODULE_PATH: 'src\'
   AUTH_MODULE_PATH: 'src\Authentication\Authentication\bin\'
   AUTH_MODULE_DLL_PATTERN: 'Microsoft.Graph.Authentication.dll'
 
 jobs:
 - job: MSGraphPSSDKGeneration
   displayName: MS Graph PS SDK Beta Generation
-  timeoutInMinutes: 600
+  timeoutInMinutes: 800
```

Comment:
wow

Reply:
This is all temporary to allow us release bug fixes for 0.9.1. @finsharp's pipeline work will eliminate the need to this long timeout time and private agent.